### PR TITLE
Adding logs for debugging

### PR DESCRIPTION
**Pull request by mchavez**
- Adding logs for debugging

### CODE REVIEW

**Comment by ggreer** [comment]
Can you include the args too?

File: pkg/mssqldb/permissions.go [hunk 1]
```diff
@@ -39,7 +40,7 @@ WHERE perms.state = 'G' OR perms.state = 'W'
 GROUP BY perms.grantee_principal_id, perms.state, principals.name, principals.type 
 ORDER BY perms.grantee_principal_id ASC 
 OFFSET @p1 ROWS FETCH NEXT @p2 ROWS ONLY`)
-
+	l.Debug("ListServerPermissions", zap.String("sql query: ", sb.String()))
```
Comment (ggreer) [inline pkg/mssqldb/permissions.go:43]: These can just be "sql query" and not "sql query: ". Zap formats these values appropriately.

Comment (mchavez) [reply pkg/mssqldb/permissions.go:43]: Fixed

File: README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ docker run --rm -v $(pwd):/out ghcr.io/conductorone/baton:latest -f "/out/sync.c
 ```
 go install github.com/conductorone/baton/cmd/baton@main
 go install github.com/conductorone/baton-sql-server/cmd/baton-sql-server@main
-baton-sql-server --dsn "server=127.0.0.1;user id=sa;password=devP@ssw0rd;port=1433" 
+baton-sql-server --dns "server=127.0.0.1;user id=sa;password=devP@ssw0rd;port=1433"
```
Comment (ggreer) [inline README.md:46]: I'm pretty sure the argument is dsn (data source name), not dns.

File: README.md [hunk 3]
```diff
@@ -84,18 +84,21 @@ Usage:
...
-      --client-secret string          The client secret used to authenticate with ConductorOne ($BATON_CLIENT_SECRET)
-      --dsn string                    The connection string for connecting to SQL Server ($BATON_DSN)
-  -f, --file string                   The path to the c1z file to sync with ($BATON_FILE) (default "sync.c1z")
-  -h, --help                          help for baton-sql-server
-      --log-format string             The output format for logs: json, console ($BATON_LOG_FORMAT) (default "json")
-      --log-level string              The log level: debug, info, warn, error ($BATON_LOG_LEVEL) (default "info")
-  -v, --version                       version for baton-sql-server
+      --client-id string       The client ID used to authenticate with ConductorOne ($BATON_CLIENT_ID)
+      --client-secret string   The client secret used to authenticate with ConductorOne ($BATON_CLIENT_SECRET)
+      --dns string             The connection string for connecting to SQL Server ($BATON_DNS)
```
Comment (ggreer) [inline README.md:94]: This needs to be dsn, not dns.

File: pkg/mssqldb/permissions.go [hunk 4]
```diff
@@ -39,7 +40,7 @@ WHERE perms.state = 'G' OR perms.state = 'W'
 GROUP BY perms.grantee_principal_id, perms.state, principals.name, principals.type 
 ORDER BY perms.grantee_principal_id ASC 
 OFFSET @p1 ROWS FETCH NEXT @p2 ROWS ONLY`)
-
+	l.Debug("ListServerPermissions", zap.String("sql query", sb.String()))
```
Comment (ggreer) [inline pkg/mssqldb/permissions.go:43]: These should log the args too.

**Review by ggreer — APPROVED** [review]

**Pull request merged**